fastlane/Fastfile

```diff
@@ -0,0 +1,127 @@
+##
+## Fastlane configuration for the Nevis Mobile Authentication SDK Android Coroutines Example application.
+## See https://docs.fastlane.tools
+##
+
+require 'java-properties'
+
+default_platform(:android)
+
+root_dir = File.expand_path(File.join(File.dirname(__FILE__), "../"))
+signing_configs_gradle = File.join(root_dir, "app/signing_configs.gradle")
+gradle_properties = File.join(root_dir, "gradle.properties")
+application_module = File.join(root_dir, "app/src/main/java/ch/nevis/exampleapp/coroutines/dagger/ApplicationModule.kt")
+android_manifest = File.join(root_dir, "app/src/main/AndroidManifest.xml")
+build_gradle = File.join(root_dir, "app/build.gradle")
+display_name = "Nevis Mobile Authentication SDK Android Coroutines Example App"
+
+def report_on_success(message)
+    slack(
+        message: "#{message}\nCheck build logs: #{ENV['CURRENT_BUILD_URL']}",
+        slack_url: ENV['SLACK_WEBHOOK_URL']
+    ) if ENV['SLACK_WEBHOOK_URL']
+end
+
+def report_on_error(message, exception)
+    slack(
+        message: "#{message}\nCheck build logs: #{ENV['CURRENT_BUILD_URL']}",
+        slack_url: ENV['SLACK_WEBHOOK_URL'],
+        success: false,
+        attachment_properties: {
+            fields: [
+                {
+                    title: "Error message",
+                    value: exception.to_s,
+                    short: false
+                }
+            ]
+        }
+    ) if ENV['SLACK_WEBHOOK_URL']
+    UI.user_error!("#{exception}")
+end
+
+platform :android do
+    desc "Build and tag the application"
+    desc ""
+    lane :main do |options|
+        begin
+            # Loading gradle.properties file.
+            properties = JavaProperties.load(gradle_properties)
```

Comment:
The coroutines app will not be uploaded to Firebase. The built app will not be available for downloading, installing.
Do we need any of the following?
- Config modification
- Singing modification

Reply:
In the case of the reactive ios example app, there are no signing config related modifications because of the reasons you mentioned.

Reply:
Removed signing config modification.